common/src/main/java/com/revenuecat/purchases/common/errors.kt

```diff
@@ -39,6 +40,9 @@ enum class BackendErrorCode(val value: Int) {
 
 fun Exception.toPurchasesError(): PurchasesError {
     return when (this) {
+        is UnknownHostException -> {
```

Comment:
Does this exception get thrown in other cases? Like when the router can't reach the internet? Looking at the docs looks like it's triggered when it can't resolve the DNS. I am just curious if you tested other poor connectivity escenarios.

Reply:
Did you get a chance to test this? The PR looks good to me, I don't want to block it with this question

Reply:
i started some on friday, wasn't able to reproduce this in another scenario... but i was gonna talk with josh for testing with another ad blocker, to see if we can more specifically identify an ad-blocking error...

so, you're not blocking :)

Reply:
I’ve been trying some testing to see if we can determine when an ad blocker is being used, but it looks like we might not get as much info as we did on iOS.

I’m able to reproduce UnknownHostException without an adblocker by using network link conditioner with a “high latency DNS” profile. The exceptions from that scenario and an ad-blocker-scenario are thrown from the same place, and I don’t see any differences between the two.

I’m not sure we can get more information without refactoring our network stack. My thought for now is that we could update the message to say something like “If you’re seeing this error in large numbers and from different users, it could be that RevenueCat requests are being blocked”. Thoughts? @taquitos